src/components/JoinUs/Work.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
+import { Link } from 'gatsby'
+import { Padding } from 'styled-components-spacing'
+import styled from 'styled-components'
+import { Row, CompensatedCol, CompensatedRow } from '../grid'
+import { H2, H3, Paragraph } from '../Typography'
+import colorLuminance from 'color-luminance'
+
+import {
+  Section,
+  FirstColumn,
+  SecondColum,
+  Separator,
+  List,
+  TitleAndBody
+} from './elements'
+
+const MAX_CASE_STUDIES = 3
+
+/* Utility function to convert an HEX color into the RGB format */
+const hexToRgb = hex => {
+  var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex)
+  return result
+    ? {
+        r: parseInt(result[1], 16),
+        g: parseInt(result[2], 16),
+        b: parseInt(result[3], 16)
+      }
+    : null
+}
+
+/* Returns the luminance (in a 0-255 range) of a color in the HEX format */
+const getColorLuminance = hexColor => {
+  const rgbValue = hexToRgb(`#${hexColor}`)
+
+  return colorLuminance(rgbValue.r, rgbValue.g, rgbValue.b)
+}
+
+const CaseStudyLink = styled(Link)`
+  background: ${props => props.bg};
+  color: ${props => props.color || props.theme.colors.white};
+  flex: 1;
+`
+
+const CaseStudy = ({ children, ...props }) => (
+  <CaseStudyLink {...props}>
+    <Padding
+      vertical={{ smallPhone: 2, smallTablet: 1.5, tablet: 3 }}
+      horizontal={{ smallPhone: 2, smallTablet: 1.5, tablet: 2 }}
+    >
+      {children}
+    </Padding>
+  </CaseStudyLink>
+)
+
+const CaseStudyWrapper = styled(Padding)`
+  display: flex;
+  width: 100%;
+`
+const CaseStudyTitle = styled(H3)`
+  font-size: ${remcalc(21)};
+  line-height: ${remcalc(24)};
+  color: ${props =>
+    props.reverse ? props.theme.colors.white : props.theme.colors.text};
+
+  ${breakpoint('largePhone')`
+    font-size: ${remcalc(17)};
+    line-height: ${remcalc(21)};
+  `};
+
+  ${breakpoint('smallTablet')`
+    font-size: ${remcalc(21)};
+    line-height: ${remcalc(24)};
+  `};
+
+  ${breakpoint('tablet')`
+    font-size: ${remcalc(26)};
+    line-height: ${remcalc(30)};
+  `};
+`
+
+const Work = ({ data: { title, list, text, someWork } }) => (
+  <Section>
+    <Row>
+      <FirstColumn>
+        <H2>{title}</H2>
+      </FirstColumn>
+      <SecondColum width={[1, 1, 1, 1, 6 / 12, 5 / 12]}>
+        <List list={list} />
+      </SecondColum>
+    </Row>
+    <Separator />
+    <TitleAndBody title="Some of our work" body={text} />
+    <CompensatedRow>
```

Comment:
You're suggesting creating a component for a 3 column layout? (not sure if I understood)

Reply:
Maybe we can revisit this idea as part of the post-merge componentisation. Might not even work. I haven't thought it through. Just an idea

Reply:
Ok, I just didn't went down that road because it's a fairly simple scenario (one row, 3 cols), not sure if a component makes sense, but yeah we can discuss it afterwards

Reply:
From our call this morning, I think you decided to do this work within this PR rather than us raising a new trello ticket for this component. Right?

Reply:
Yes!